config/jest/setup.js

```diff
@@ -2,8 +2,8 @@
 import './raf';
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import toJson from 'enzyme-to-json';
-import * as matchers from './matchers'
+import * as matchers from './matchers';
+import 'jest-styled-components';
```

Comment:
Globally importing `jest-styled-components` instead of in every test - this fixed the "could not resolve SC internals" errors